src/components/app/App.jsx

```diff
@@ -0,0 +1,110 @@
+// /* eslint-disable max-len */
+import React, { useReducer } from 'react';
+
+{/* ---------------------------------------------------- */}
+
+
+const initialState = {
+  color: '#ff0000',
+  before: [],
+  after: []
+};
+
+function reducer(state, action) {
+  switch(action.type) {
+    case 'COLOR_CHANGE': {
+      const before = [...state.before, state.color];
```

Comment:
this is not the proper syntax. 
https://redux.js.org/tutorials/fundamentals/part-3-state-actions-reducers